app/actions/index.js

```diff
@@ -41,9 +41,7 @@ export function sourcesSearch (terms, plugins) {
     Object.assign(searchResults, plugins.musicSources[i].search(terms));
   }
 
-  return {
-    type: SOURCES_SEARCH
-  };
+  return { type: undefined };
```

Comment:
I think this will introduce a bug.

Reply:
Nvm, actually this function can probably be deleted altogether.

Reply:
Yeah, I wasn't sure what it was going to be user for so I just exposed the bug.